en/deploy/README.md

```diff
@@ -130,32 +130,64 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
+## Configuring our site on PythonAnywhere
 
-<img src="images/pythonanywhere_bash_console.png" alt="pointing at Other: Bash in Start a new Console" />
+Go back to the main PythonAnywhere Dashboard by clicking on the logo, and choose the option to start a "Bash" console – that's the PythonAnywhere version of a command line, just like the one on your computer. 
+
+<img src="images/pythonanywhere_bash_console.png" alt="Pointing at Bash in the New Console section" />
 
 > **Note** PythonAnywhere is based on Linux, so if you're on Windows, the console will look a little different from the one on your computer.
 
-Let's pull down our code from GitHub and onto PythonAnywhere by creating a "clone" of our repo. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
+Deploying a web app on PythonAnywhere involves pulling down your code from GitHub, and then configuring PythonAnywhere to recognise it and start serving it as a web application.  There are manual ways of doing it, but PythonAnywhere provide a helper tool that will do it all for you. Let's install it first:
+
+{% filename %}PythonAnywhere command-line{% endfilename %}
+```
+$ pip3.6 install --user pythonanywhere
+```
+
+That should print out some things like `Collecting pythonanywhere`, and eventually end with a line saying `Successfully installed (...) pythonanywhere- (...)`.
+
+Now we run the helper to automatically configure our app from GitHub. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ git clone https://github.com/<your-github-username>/my-first-blog.git
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
 ```
 
-This will pull down a copy of your code onto PythonAnywhere. Check it out by typing `tree my-first-blog`:
+As you watch that running, you'll be able to see what it's doing:
+
+- Downloading your code from GitHub
+- Creating a virtualenv on PythonAnywhere, just like the one on your own PC
+- Updating your settings file with some deployment settings
+- Setting up a database on PythonAnywhere using the "migrate" command
+- Setting up your static files (we'll learn about these later)
+- And configuring PythonAnywhere to serve your web app via its API
+
+On PythonAnywhere all those steps are automated, but they're the same steps you
+would have to go through with any other server provider.  The main thing to notice
+right now is that your database on PythonAnywhere is actually totally separate from
+your database on your own PC -- that means it can have different posts and admin accounts.
+
+Check out your code on PythonAnywhere by typing `tree`:
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ tree my-first-blog
-my-first-blog/
+$ tree 
+yourusername.pythonanywhere.com/
```

Comment:
agreed.  I was just adapting from the existing `tree` listing.  Options:

* remove the tree command altogether
* `tree *.pythonanywhere.com` should pick up on any username
* just `tree` with no arguments will probably be fine!  since the blog repo will be the only folder in their account, that's all they would see, apart from our Readme.txt

that last seems like the easiest?  I'll make that change unless y'all have another preference.

Reply:
oh, i see i had already implemented solution 3?  so there is actually no copy-paste error possible here...